src/vm/eepolicy.cpp

```diff
@@ -1117,6 +1153,9 @@ void EEPolicy::LogFatalError(UINT exitCode, UINT_PTR address, LPCWSTR pszMessage
 
     _ASSERTE(pExceptionInfo != NULL);
 
+    // Log exception to StdErr
```

Comment:
Is this going to kick for regular unhandled exceptions? I believe we have a good message for regular unhandled exceptions already. This won't make them better.

Maybe this needs to be in a slightly different place so that it kicks in for failfasts only.